### DISCUSSION TITLE
[A-L] (8 maggio 2024) Esercizio 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/progettazione-di-algoritmi#68] [Esercitazioni A-L]
![image](https://github.com/sapienzastudentsnetwork/progettazione-di-algoritmi/assets/40034429/b0f8cbd3-691f-4a62-b2f1-6f8d41d10ec6)

**Comment by daromacarlo** [comment]
![immagine](https://github.com/user-attachments/assets/81fcd3fe-ef99-4155-913a-f3be6bd5a958)
